docs/controls/file-pickers/node/theming-customization.md

```diff
@@ -0,0 +1,89 @@
+---
+author: natalieethell
+ms.author: natalieethell
+ms.date: 10/03/2018
+ms.topic: conceptual
+title: Add custom theme to file browser component
+---
+
+# Theming customization
+
+You can give your File Browser a custom theme by utilizing Fabric's `Customizer` component.
```

Comment:
There are a number of Fabric's so we might want to clarify and use `Office UI Fabric React` throughout.